### PR TITLE
chore: Update vm-console-proxy to v0.5.0

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Updated `vm-console-proxy` to version `v0.5.0`, and changed code to support multiple `ClusterRoles`.

**Release note**:
```release-note
Updated vm-console-proxy to version v0.5.0.
```

### CODE REVIEW

File: internal/vm-console-proxy-bundle/bundle.go [hunk 1]
```diff
@@ -142,5 +144,10 @@ func validateBundle(bundle *Bundle) error {
 	if len(missingFields) > 0 {
 		return fmt.Errorf("bundle is missing these objects: %v", missingFields)
 	}
+
+	if len(bundle.ClusterRoles) != 2 {
```
Comment (0xFelix) [inline internal/vm-console-proxy-bundle/bundle.go:148]: Do we need to hardcode this check here?

Comment (akrejcir) [reply internal/vm-console-proxy-bundle/bundle.go]: I've just removed the condition. Making the code more generic is maybe a task for a future PR.

**Review by 0xFelix — COMMENTED** [review]
/approve

Thanks looks good to me.

**Comment by jcanocan** [comment]
Great work @akrejcir !
/lgtm

**Pull request merged**